app/static/js/currency.js

```diff
@@ -0,0 +1,11 @@
+function Convert(n1, n2, conversion_factor) {
+    // id's for currency input boxes
+    id1 = "cb" + n1; 
+    id2 = "cb" + n2;
+
+    // getting the value of the input box that just got filled
+    var input_box = document.getElementById(id1).value;
+
+    // updating the other input box after conversion
+    document.getElementById(id2).value = ((input_box * conversion_factor).toFixed(2));
+}
```

Comment:
ES6/formatting updates:

```suggestion
const convert = (n1, n2, conversionFactor) => {
    // id's for currency input boxes
    let id1 = "cb" + n1; 
    let id2 = "cb" + n2;

    // getting the value of the input box that just got filled
    let inputBox = document.getElementById(id1).value;

    // updating the other input box after conversion
    document.getElementById(id2).value = ((inputBox * conversionFactor).toFixed(2));
}
```

Reply:
Um I am not so well versed with js.  Could you explain as to why we would take `const convert = ` and then `=>`?

Reply:
The "fat arrow" syntax is just part of ES6 -- [see here](https://www.javascripttutorial.net/es6/javascript-arrow-function/).

Declaring as `const` makes the function immutable, which helps avoid any issues (albeit unlikely) where other parts of the js code change this function.